kernel/packages/shared/social/hotScenes.ts

```diff
@@ -4,10 +4,17 @@ import { StoreContainer } from 'shared/store/rootTypes'
 import { fetchSceneIds } from 'decentraland-loader/lifecycle/utils/fetchSceneIds'
 import { fetchSceneJson } from 'decentraland-loader/lifecycle/utils/fetchSceneJson'
 import { SceneJsonData } from 'shared/types'
+import { reportScenesFromTiles } from 'shared/atlas/actions'
 import { getSceneNameFromAtlasState, postProcessSceneName } from 'shared/atlas/selectors'
 
 declare const globalThis: StoreContainer
 
+declare const window: {
```

Comment:
> You could use `UnityInterfaceContainer` from `dcl.ts` here instead.

Wouldn't that end up in a circular dependency issue?

Reply:
Indeed, good catch.

Reply:
We can't move `UnityInterfaceContainer` to a new file to solve the circular dependency and be able to use it nicely?

Reply:
> We can't move `UnityInterfaceContainer` to a new file to solve the circular dependency and be able to use it nicely?

if we choose to go that way, we should create another PR for that... cause, as far as I can see, we have multiple scripts doing this same thing, so we should create a PR to fix'em all 😄

Reply:
O.K. by me, but if the improvement is feasible without much hassle and makes sense, pretty please let's open another PR and fix it. I'm a bit concerned about not improving anything because of bureaucracy.